docs/getting-started/introduction.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Overview
+---
+
+## What is Tach?
+
+Tach allows you to control dependencies between your Python modules.
+Modules can also define an explicit public interface to prevent deep coupling.
+
+This creates a decoupled, modular architecture, which makes maintenance and development easier.
+
+If a module tries to import from another module that is not listed as a dependency, Tach will report an error.
+
+When a module is in ['strict mode'](usage/strict-mode), if another module tries to import from it without using its public interface, Tach will report an error.
+
+Tach is a CLI tool, and is ideal for pre-commit hooks and CI checks.
+
+Tach is:
+
+- 🌎 Open source
+- 🐍 Installable via pip
+- 🔧 Able to be adopted incrementally
+- ⚡ Implemented with no runtime impact
+- ♾️ Interoperable with your existing systems (cli, hooks, ci, etc.)
+
+## Commands
+
+- [`tach mod`](usage/commands#tach-mod) - Interactively define module boundaries.
+- [`tach check`](usage/commands#tach-check) - Check that boundaries are respected.
+- [`tach sync`](usage/commands#tach-sync) - Sync constraints with the actual dependencies in your project.
+- [`tach show`](usage/commands#tach-show) - Visualize your project's dependency graph.
+- [`tach report`](usage/commands#tach-report) - Generate a dependency report for a file or module.
+- [`tach install`](usage/commands#tach-install) - Install Tach into your development workflow (e.g. pre-commit)
```

Comment:
While you were doing this we added `tach test`, which added an entry to this list (you can find it on main)

Reply:
Added tach test back